src/sentry/plugins/providers/integration_repository.py

```diff
@@ -145,7 +139,7 @@ def create_repository(
                 except IntegrationError:
                     pass
 
-                raise RepoExistsError
+                metrics.incr("sentry.integration_repo_provider.repo_exists")
```

Comment:
We should keep the exception, but ignore it in a try catch around the function call.

Reply:
what is the difference between removing it and ignoring it?